### PR TITLE
Fix README and license

### PR DESCRIPTION
**Pull request by sbesson**
While reviewing #30, I started reviewing the README and updated this repository as well as the static website footer to license it under CC-BY 4.0 license (similarly to the OME website).

### CODE REVIEW

**Comment by manics** [comment]
Since you're touching this file, any chance you could change all http to https too?

**Comment by manics** [comment]
Thanks, changes look good, let's check it after the next merge build.

**Comment by joshmoore** [comment]
@manics : what checking did you mean with "after the next merge build"?

**Comment by manics** [comment]
https://snoopycrimecop.github.io/idr.openmicroscopy.org/

File: _includes/footer-idr.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
         <div class="callout large secondary">
             <div class="row">
                 <div class="row small-up-2 medium-up-5 large-up-5">
-                    <div class="column"><a href="http://www.openmicroscopy.org/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/ome-logo-200.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="OME"></a></div>
+                    <div class="column"><a href="https://www.openmicroscopy.org/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/ome-logo-200.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="OME"></a></div>
                     <div class="column"><a href="http://www.eurobioimaging.eu/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/eurobioimaging_logo.gif" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="Euro-Bioimaging"></a></div>
                     <div class="column"><a href="http://www.eurobioimaging.eu/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/globalbioimaging_logo.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="Global-Bioimaging"></a></div>
```
Comment (manics) [inline _includes/footer-idr.html]: Just noticed, should global-bioimaging link to http://www.eurobioimaging.eu/global-bioimaging instead of the top level?

**Comment by joshmoore** [comment]
K. HTTPS links work as does the new CC link. :+1:

File: _includes/footer-idr.html (hunk 1 above)
Comment (joshmoore) [reply _includes/footer-idr.html]: I'd defer to @jrswedlow

Comment (jrswedlow) [reply _includes/footer-idr.html]: At the moment, yes, that is the correct link.  You can shoot the messenger, but it won't change anything.

**Pull request merged**